datasets/Hamilton_2022/Hamilton_2022.json

```diff
@@ -0,0 +1,20 @@
+{
+    "dataset_id": "Hamilton_2022",
+    "url": "",
```

Comment:
Educated guess?

Reply:
I have sent out an email requesting a link. I'll update with new info

Reply:
So, what happens when a dataset doesn't have any other page than the OSF page? @J535D165